src/Data/Managers/V2/AddUser/useAddUser.ts

```diff
@@ -39,7 +40,9 @@ export const useAddUser = () => {
       body: JSON.stringify(orderBody),
       headers,
     });
-    handleAuthorization(response);
+    if (response.status === 401) {
```

Comment:
Why do this everywhere manually? Either there is a method to hook into fetch when the status code is 401 or create a function that does this.

or use the mechanism of before and create a function that handles the response.

Reply:
Moved it into handleErrors hook